### PR TITLE
 Add Job ssh for kube-runtime. Create ssh key secret in cluster-config

### PR DESCRIPTION
**Pull request by wangdian**
Add Job ssh for kube-runtime. 
Create ssh key secret in cluster-config

### CODE REVIEW

File: src/kube-runtime/src/init/sshd.sh [hunk 1]
```diff
@@ -20,22 +20,69 @@
...
-        echo "no private key or port provided" >&2
-    else
-        mkdir -p $CONFIG_DIR
+function prepare_ssh()
+{
+  mkdir /root/.ssh
+  touch /root/.ssh/authorized_keys
+  chmod 600 /root/.ssh/authorized_keys
+
+  mkdir -p /var/run/sshd
```
Comment (abuccts) [inline src/kube-runtime/src/init/sshd.sh]: do not need?

File: src/kube-runtime/src/init/sshd.sh [hunk 2]
```diff
@@ -20,22 +20,69 @@
...
+    if [ -z "$PAI_CONTAINER_SSH_PORT" ] ; then
+        echo "no ssh port provided" >&2
+    else
+        prepare_ssh
+        prepare_job_ssh
+        prepare_user_ssh
+        start_ssh
     fi
 else
     echo "no sshd binary found" >&2
```
Comment (abuccts) [inline src/kube-runtime/src/init/sshd.sh]: install openssh server for user if there's no sshd in image?

File: src/rest-server/src/models/v2/job/k8s.js [hunk 3]
```diff
@@ -255,6 +255,11 @@ const generateTaskRole = (taskRole, labels, config) => {
                   name: 'host-log',
                   mountPath: '/usr/local/pai/logs',
                 },
+                {
+                  name: 'job-ssh-secret-volume',
+                  readOnly: true,
+                  mountPath: '/etc/ssh-secret',
```
Comment (abuccts) [inline src/rest-server/src/models/v2/job/k8s.js]: maybe mount to `/usr/local/pai/secrets`?

File: src/kube-runtime/src/init/sshd.sh (hunk 1 above)
Comment (wangdian) [reply src/kube-runtime/src/init/sshd.sh]: Better have this in case the files are not created.

File: src/kube-runtime/src/init/sshd.sh (hunk 2 above)
Comment (wangdian) [reply src/kube-runtime/src/init/sshd.sh]: Ok

File: src/rest-server/src/models/v2/job/k8s.js (hunk 3 above)
Comment (wangdian) [reply src/rest-server/src/models/v2/job/k8s.js]: I think mount to /etc/ssh-secret is fine because I don't think we should use ssh keys elsewhere.

Comment (wangdian) [reply src/rest-server/src/models/v2/job/k8s.js]: Changed mountPath to /usr/local/pai/ssh-secret. Although I think user should not use this key elsewhere.

**Pull request merged**